packages/block-library/src/rss/index.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Server-side rendering of the `core/rss` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the rss error message.
+ *
+ * @param string $message RSS error message.
+ * @return string|void Void if renders on frontend.
+ */
+function render_rss_error_message( $message ) {
+	return '<div class="components-placeholder"><div class="notice notice-alt notice-error">' . $message . '</div></div>';
+}
+
+/**
+ * Renders the `core/rss` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content with received rss items.
+ */
+function render_block_core_rss( $attributes ) {
+
+	if ( ! isset( $attributes['feedURL'] ) ) {
+		return;
+	}
+
+	$url = filter_var( $attributes['feedURL'], FILTER_VALIDATE_URL );
```

Comment:
Can‘t use filter_var because it might not be available. Use sanitization and escaping functions built into core of needed. I don‘t think it‘s necessary here at all. fetch_feed() should handle that already, so I think you should just remove this line

Reply:
I removed this line and will do some tests with different inputs.